index.js

```diff
@@ -6,17 +6,19 @@
 var CSSescape = require('css.escape')
 
 exports.hash = function hash (h, options) {
-  if (!h) return
   options = options || {}
+  var cb = options.callback || function () {}
+  // There's no hash to scroll to, so "success", we did it.
+  if (!h) return void cb(false)
```

Comment:
I would prefer just `cb()` here - passing `undefined`.

Reply:
Yeah, I wavered back and forth on that.